src/services/rfqm_service.ts

```diff
@@ -209,14 +214,17 @@ const RFQM_PROCESS_JOB_LATENCY = new Summary({
 });
 const PRICE_DECIMAL_PLACES = 6;
 
+const INITIAL_MAX_PRIORITY_FEE_PER_GAS = new BigNumber(4);
```

Comment:
It seems like this might be in gwei but ends up being used as if it's wei.

Reply:
Also seems really high (this screenshot is from a fairly high gas environment)
<img width="793" alt="Screen Shot 2021-11-03 at 4 16 23 PM" src="https://user-images.githubusercontent.com/5778036/140200911-895bbcd4-da4d-45cb-86b3-7c6dbad8c4a0.png">